examples/visual-tests/amp-story/amp-story-page-attachment.js

```diff
@@ -224,4 +224,38 @@ module.exports = {
         '.i-amphtml-story-page-attachment-domain-label',
     ]);
   },
+
+  'form presence - the keyboard should close upon attachment dismissal': async (
+    page,
+    name
+  ) => {
+    const url = await page.url();
+    const pageID = 'attachment-with-form';
+    await page.goto(`${url}#page=${pageID}`);
+    await page.waitForSelector(
+      `amp-story-page#${pageID}[active][distance="0"]`
+    );
+
+    // Open the page attachment.
+    await page.waitForSelector(
+      `amp-story-page#${pageID} .i-amphtml-story-inline-page-attachment-chip`
+    );
+    await page.tap(
+      `amp-story-page#${pageID} .i-amphtml-story-inline-page-attachment-chip`
+    );
+
+    // Tap the input element to cause the soft keyboard to appear.
```

Comment:
It could be good to make this more descriptive. A developer might think there is a special amp keyboard that's supposed to always appear. It could be something like:
"Tap the input element, this causes the soft keyboard to appear on mobile."